src/aiida/manage/configuration/config.py

```diff
@@ -780,7 +781,8 @@ def _atomic_write(self, filepath=None):
                 os.umask(umask)
 
             handle.flush()
-            os.rename(handle.name, self.filepath)
```

Comment:
Does os.rename works differently on windows?

Reply:
Ah right probably on windows rename really only allows renaming while we are moving here potentially a file.